deep_quoridor/src/agents/agent.py

```diff
@@ -4,5 +4,26 @@ class Agent:
     Given a game state, the agent should return an action.
     """
 
+    agents = {}
```

Comment:
nit: I would move this code to `__init__.py`. That way, when you look at the directory structure, you don't get confused by one file which is not an agent but an interface. I've seen this pattern in other bigger projects too.